src/pipx/pipx_metadata_file.py

```diff
@@ -35,6 +35,7 @@ class PackageInfo(NamedTuple):
     apps_of_dependencies: List[str]
     app_paths_of_dependencies: Dict[str, List[Path]]
     package_version: str
+    suffix: str = ""
 
 
 class PipxMetadata:
```

Comment:
For any change to the pipx metadata format, line 66 `self._pipx_metadata_version: str = "0.1"` needs to be updated with an incremented version string.

I think a good approach would be a new module global `PIPX_METADATA_VERSION = "0.2"` at the top, and change line 66 to `self._pipx_metadata_version: str = PIPX_METADATA_VERSION`

Reply:
@itsayellow I have made the changes but using a slightly different approach. I reckon since this version is tied to the class, I have made it a class variable. Should ahve the same desired effect. Happy to change it to your suggestion if you think that is better.

Reply:
Can you use something more like `__METADATA_VERSION__` instead of `__VERSION__`?  I want to make sure nobody confuses this with the version of pipx, it is specifically for the format of the metadata we write to file.

Reply:
@itsayellow consider it done! :)